README.old.md

```diff
@@ -0,0 +1,2 @@
+# cast-le-frontend
+The frontend repository for the cast-le project
```

Comment:
Since we don't want two README files, could we move this back into the normal README and delete this one?

Reply:
fixed